scripts/PortableToFullPdb.ps1

```diff
@@ -32,45 +32,45 @@ function Locate-PdbConverterTool
 }
 
 function ConvertPortablePdbToWindowsPdb
-{	
+{
     $allPdbs = Get-ChildItem -path $TP_OUT_DIR\$Configuration *.pdb -Recurse | % {$_.FullName}
     $portablePdbs = New-Object System.Collections.Generic.List[System.Object]
-	
+
     foreach($pdb in $allPdbs)
     {
-	# First four bytes should be 'BSJB' for portable pdb
-	$bytes = [char[]](Get-Content $pdb -Encoding byte -TotalCount 4) -join ''
-	
-	if( $bytes -eq "BSJB")
-	{
-		$portablePdbs.Add($pdb)
-	}
+    # First four bytes should be 'BSJB' for portable pdb
```

Comment:
This looks off, does it look okay in editor?

Reply:
I'll investigate.

Reply:
Indeed, it was off. I have re-applied the format.